src/java.base/share/classes/jdk/internal/access/JavaLangAccess.java

```diff
@@ -525,16 +528,49 @@ public interface JavaLangAccess {
     /**
      * Re-enables a virtual thread for scheduling. If the thread was parked then
      * it will be unblocked, otherwise its next attempt to park will not block
+     *
      * @param thread the virtual thread to unpark
-     * @throws IllegalArgumentException if the thread is not a virtual thread
+     * @throws IllegalArgumentException   if the thread is not a virtual thread
      * @throws RejectedExecutionException if the scheduler cannot accept a task
      */
     void unparkVirtualThread(Thread thread);
 
     /**
      * Creates a new StackWalker
      */
-    StackWalker newStackWalkerInstance(Set<StackWalker.Option> options,
-                                       ContinuationScope contScope,
-                                       Continuation continuation);
+    StackWalker newStackWalkerInstance(Set<StackWalker.Option> options, ContinuationScope contScope, Continuation continuation);
```

Comment:
Is the formatting change in this file (this line and above) accidental done by IDE?

I expect the change should just be the addition of the new methods below.